ajax/libs/FormChimp/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "FormChimp",
+  "filename": "jquery.formchimp.min.js",
+  "description": "A customizable MailChimp ajax plugin for jQuery",
+  "author": {
+    "name": "Fabio Quarantini",
+    "url": "http://www.fabioquarantini.com",
+    "twitter": "@febba"
+  },
+  "homepage": "http://www.fabioquarantini.com/formchimp/",
+  "keywords": [
+    "FormChimp",
+    "MailChimp",
+    "jQuery plugin",
+    "ajax"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabioquarantini/formchimp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/fabioquarantini/formchimp.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.formchimp*.+(js|css|map)"
```

Comment:
Sorry for carelessness, 
In this case, we need to use `"jquery.formchimp!(*-[0-9])*.+(js|css|map)"` to ignore `jquery.formchimp.min-0.1.0.js` in v1.0.0.
Thank you.